product_docs/docs/jdbc_connector/42.5.1.2/06_executing_sql_commands_with_executeUpdate().mdx

```diff
@@ -30,41 +34,45 @@ The example that follows shows using the `executeUpdate()` method to add a row t
 
 
 ```java
-    public void updateEmployee(Connection con)
+    public void addOneEmployee(Connection con)
     {
-      try
-      {
-        int rowcount = stmt.executeUpdate("INSERT INTO "
-          + "emp(empno, ename) VALUES(6000,'Jones')");
-        System.out.println("");
-        System.out.println("Success - "+rowcount+
-          " rows affected.");
-      } catch(Exception err) {
-        System.out.println("An error has occurred.");
-        System.out.println("See full details below.");
-        err.printStackTrace();
-      }
+        try
+        {
+            Statement stmt=con.createStatement();
```

Comment:
Best practice is to use try-with-resources here: 
eg.
```
    try (Statement stmt = con.createStatement()) 
    {
        ....
```

Reply:
Changed